integration/unknown-fields/unknown-fields-test.ts

```diff
@@ -382,7 +382,7 @@ describe("unknown-fields", () => {
             },
           ],
         },
-        "ccEnableArenas": false,
+        "ccEnableArenas": true,
```

Comment:
This changed because the file imports at proto2 message, hence the correct default value is being picked up.